_pytest/python.py

```diff
@@ -43,6 +44,19 @@ def _format_args(func):
     def _format_args(func):
         return inspect.formatargspec(*inspect.getargspec(func))
 
+class _GotAllAttrs():
```

Comment:
That'd probably be `(type, types.ClassType)` for Python2 and `type` for Python 3. I'm fine with both solutions.